src/client/lib/utils.js

```diff
@@ -17,7 +17,7 @@ export var serialize = (obj) => JSON.stringify(obj || {});
  * @params {String} str: Object representation string.
  * @return {Object} The deserialized Object.
  */
-export var deserialize = (str) => JSON.parse(str || '{}');
+export let deserialize = (str) => JSON.parse(str || '{}');
```

Comment:
'export' is only available in ES6 (use esnext option).<br>'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>'arrow function syntax (=>)' is only available in ES6 (use esnext option).